policyengine_uk/parameters/gov/dfe/extended_childcare_entitlement/expense_rate.yaml

```diff
@@ -0,0 +1,25 @@
+description: The Department for Education sets hourly rates to convert childcare hours into expenses for extended childcare entitlement. These rates vary by child's age and are used to calculate the funding provided to childcare providers.
+metadata:
+ type: single_amount
+ threshold_unit: year
+ amount_unit: currency-GBP
+ label: Extended childcare entitlement hourly funding rates by age
+ reference:
+   - title: different rates for extended childcare entitlement, please follow the links
```

Comment:
Follow the style for reference titles please. Title should be the title of the document, nothing else (also capitalise)